lldb/include/lldb/Expression/Expression.h

```diff
@@ -96,6 +96,41 @@ class Expression {
                                  ///invalid.
 };
 
+/// Holds parsed information about a function call label that
+/// LLDB attaches as an AsmLabel to function AST nodes it parses
+/// from debug-info.
+///
+/// The format being:
+///
+///   <prefix>:<mangled name>:<module id>:<DIE id>
+///
+/// The label string needs to stay valid for the entire lifetime
+/// of this object.
+struct FunctionCallLabel {
+  /// Mostly for debuggability.
+  llvm::StringRef m_pubname;
+
+  lldb::user_id_t m_die_id;
```

Comment:
Technically we could just encode the mangled name and module ID. Since we're just doing a mangled name lookup within the module anyway.